lib/airbrake/rails3_tasks.rb

```diff
@@ -59,6 +59,11 @@ def wait_for_threads
     class ActionDispatch::DebugExceptions; def call(env); @app.call(env); end; end
     class ActionDispatch::ShowExceptions; def call(env); @app.call(env); end; end
 
+    # Override BetterErrors middleware if present
```

Comment:
When we run `rake airbrake:test` task, we want clean `STDOUT` (only our stuff) and that's the reason we override few middlewares which can add their own stack trace.